.ci/test.sh

```diff
@@ -97,12 +97,12 @@ fi
 if [[ $TASK == "lint" ]]; then
     mamba create -q -y -n $CONDA_ENV \
         ${CONDA_PYTHON_REQUIREMENT} \
-        'cmakelint>=1.4.2' \
-        'cpplint>=1.6.0' \
-        'matplotlib-base>=3.8.3' \
-        'mypy>=1.8.0' \
-        'pre-commit>=3.6.0' \
-        'pyarrow>=6.0' \
+        'cmakelint>=1.4.3' \
+        'cpplint>=1.6.1' \
```

Comment:
```suggestion
        'cpplint>=1.6.0' \
```

Guess 1.6.1 is not on conda-forge yet: https://anaconda.org/conda-forge/cpplint